### DISCUSSION TITLE
Where can I label papers with alphabetical author order?

### DISCUSSION THREAD

**Discussion by Ruyc** [alshedivat/al-folio#1726] [Q&A]
Hi I am working on both theory and application, so some of my papers have alphabetical author order. I want to label those papers somewhere to claim their authors are listed by such rules. Any idea where and how to put these claims?

One way I can think about is add the claims/explanations in abbrs.. But the text will be too long and exceed the place. 

I appreciate any suggestions! Thanks so much!!

**Comment by george-gca** [comment]
The best way I could think would be adding something, like a tag or an emoticon, indicating such thing, then put a legend somewhere.

**Reply by Ruyc** [reply]
Thanks! I will see how can I add this

**Reply by Ruyc** [reply]
But also equal contribution...maybe I need to add some superscripts on the authors..

**Comment by george-gca** [comment]
Is all that information really necessary when seeing an overview of the papers? I mean, it is already written in the papers somehow, maybe it is not something relevant outside. Also it will be a lot of information being displayed.

**Reply by Ruyc** [reply]
Maybe not that necessary.. But my name is easy to be displayed as the first author (under alphabetical order). I just want to clarify that it is alphabetical in order to give my coauthors enough credits  : )

**Reply by Ruyc** [reply]
Maybe I am being too sensitive.. you know, I do not want to take advantage of the rule since not all my papers are using the same order. 
Thx a lot for providing such useful advice! I will think about it again

**Reply by george-gca** [reply]
I think it's OK to relax about that, at least in the paper overview. Generally speaking this kind of information is already presented in the paper, so I would avoid it in the overview just for the sake of not filling it with too much information, which could distract from the main reason that overview is there.
